selenium-system-test/src/test/scala/htsstride/steps/SCRCustomerJourneySteps.scala

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package htsstride.steps
+
+import htsstride.browser.Browser
+import htsstride.pages._
+import htsstride.pages.eligibility._
+import htsstride.utils.NINOGenerator
+import htsstride.utils.CustomerDetails.validCustomerDetails
+
+class SCRCustomerJourneySteps extends Steps with NINOGenerator {
+
+  Given("^the operator is logged in with SCR clearance$") {
+    StrideSignInPage.authenticateSCROperator()
+  }
+
+  When("^the operator fills in the applicant's details$") {
+    SCRCustomerEligiblePage.fillInSCRDetails(validCustomerDetails)
+  }
+
+  And("^check the applicant's details are correct$") {
+    CheckCustomersDetailsPage.submitAndCheck()
+  }
+
+  When("^the operator enters the first name as null$") {
+    SCRCustomerEligiblePage.fillInSCRDetails(validCustomerDetails.copy(firstName = Some("")))
+  }
+
+  Then("^the enter a first name error message appears$") {
+    Browser.checkCurrentPageIs(ErrorSCREligiblePage)
+    ErrorSCREligiblePage.findErrorMessageList() shouldBe Some("Enter a first name")
+    ()
+  }
+
+  When("^the operator enters a last name with over 300 characters$") {
+    val surname = List.fill(301)("a").mkString("")
+    SCRCustomerEligiblePage.fillInSCRDetails(validCustomerDetails.copy(lastName = Some(surname)))
+  }
+
+  Then("^the last name over the limit error message appears$") {
+    Browser.checkCurrentPageIs(ErrorSCREligiblePage)
+    ErrorSCREligiblePage.findErrorMessageList() shouldBe Some("Last name must be 300 characters or less")
+    ()
+  }
+
+  When("^the operator enters a date of birth in the future$") {
+    SCRCustomerEligiblePage.fillInSCRDetails(validCustomerDetails.copy(dobYear = Some("2100")))
+  }
+
+  Then("^the date of birth must be in the past error message appears$") {
+    Browser.checkCurrentPageIs(ErrorSCREligiblePage)
+    ErrorSCREligiblePage.findErrorMessageList() shouldBe Some("Date of birth must be in the past")
+    ()
+  }
+
+  When("^the operator enters a month as 0 and the year as null$") {
+    SCRCustomerEligiblePage.fillInSCRDetails(validCustomerDetails.copy(dobMonth = Some("0"), dobYear = Some("")))
```

Comment:
null would technically be `.copy(dobYear = None)`. Having `.copy(dobYear = "")` would enter a defined year, but it would just be empty